src/azure-cli/azure/cli/command_modules/synapse/manual/_help.py

```diff
@@ -155,6 +155,10 @@
     text: |-
         az synapse spark pool update --name testpool --workspace-name testsynapseworkspace --resource-group rg \\
         --package-action Add --package package1.jar package2.jar
+  - name: Update the Spark pool's configuration file.
+    text: |-
+        az synapse spark pool update --name testpool --workspace-name testsynapseworkspace --resource-group rg \\
+        --spark-config-file-path 'path/configfile.txt'
```

Comment:
@path/configfile? please double check, thanks

Reply:
Hi @zesluo , I think only JSON file need to add `@` at the front of the path. https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md#best-practice-use-file-input-for-json